Sources/MapboxDirections/MapMatching/MapMatchingResponse.swift

```diff
@@ -4,15 +4,39 @@ import FoundationNetworking
 #endif
 import Turf
 
+/**
+ A `MapMatchingResponse` object is a structure that corresponds to a map matching response returned by the Mapbox Map Matching API.
+ */
 public struct MapMatchingResponse: ForeignMemberContainer {
+    /**
+     Unrecognized properties that are documented as "beta" properties in the Mapbox Map Matching API response.
+     */
```

Comment:
This property is already documented by way of `ForeignMemberContainer`, so unless there’s something unique about foreign members in the context of a top-level response object, we can omit the documentation comment for this particular property. (Same for `RouteResponse.foreignMembers`.)

https://github.com/mapbox/mapbox-directions-swift/blob/3e0d1ac37aca0489965028e4521623531a02cb7d/Sources/MapboxDirections/Extensions/ForeignMemberContainer.swift#L57-L61